RELEASE.md

```diff
@@ -2,34 +2,82 @@
 
 The release workflow has the following steps, which are scripted.  All
 commands below take additional options, see `./release --help` for
-details.  Publishing artefacts to pypi requires write permission to
-pypi.
+details.
+
+The big picture process:
+
+1. Development release, generating a `datawire-quarkdev` package from the
+   develop branch.
+2. The stable release, landing the `develop` branch on `master` and then a
+   generating `datawire-quark` package.
+
+## Requirements ##
+
+* Tools required are `docker`, `pip`, `virtualenv`, and a not totally ancient
+  `git`.
+* Publishing artifacts to pypi requires write permission to PyPI.
+* Publishing artifacts to DockerHub requires a DockerHub login and that you be
+  part of the `datawire` organization there.
+* A GPG key with which to sign the release.
```

Comment:
Maybe mention https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work to save a Google search?